pkg/agent/endpoints/sdsv2/handler_test.go

```diff
@@ -28,7 +29,7 @@ import (
 )
 
 var (
-	tdBundle = bundleutil.BundleFromRootCA("spiffe://domain.test", &x509.Certificate{
+	tdBundle = bundleutil.BundleFromRootCA(spiffeid.RequireTrustDomainFromString("spiffe://domain.test"), &x509.Certificate{
```

Comment:
nit: even though RequireTrustDomainFromString can parse the trust domain out of the ID, i think it is clearer in intent to just pass the trust domain:
```suggestion
	tdBundle = bundleutil.BundleFromRootCA(spiffeid.RequireTrustDomainFromString("domain.test"), &x509.Certificate{
```